index.html

```diff
@@ -3,8 +3,8 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="data.js"></script>
-  <script src="index.js"></script>
+  <script defer src="data.js"></script>
```

Comment:
For the `data.js` file you don't actually need defer, since none of the code in that file references any DOM elements. It won't break by using defer - just wanted to point that out!